Clients/src/presentation/pages/Assessment/index.css

```diff
@@ -0,0 +1,28 @@
+
+.header{
+    font-weight: bold;
+}
+
+.grid-container{
+    grid-template-columns: auto auto auto;
```

Comment:
Hi Matt, Instead of using auto auto auto, use repeat(3, 1fr) for equal column widths.